packages/richtext-lexical/src/utilities/initLexicalFeatures.ts

```diff
@@ -76,11 +79,7 @@ export function initLexicalFeatures(args: Args): {
     // as well, as it already called feature.generateSchemaMap for each feature.
     // We will check for the existance resolvedFeature.generateSchemaMap to skip unnecessary loops for constructing featureSchemaMap, but we don't run it here
     if (resolvedFeature.generateSchemaMap) {
-      const featureSchemaPath = [
-        ...args.schemaPath.split('.'),
-        'lexical_internal_feature',
-        featureKey,
-      ].join('.')
+      const featureSchemaPath = `${args.schemaPath}.lexical_internal_feature.${featureKey}`
```

Comment:
Perf improvement. Why .split and .join, if we can just manually append to the schemaPath string.